frontend/src/app/components/employee/emp-absences-index/emp-absences-index.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, OnInit } from "@angular/core";
+import { MatDialog, MatDialogConfig } from "@angular/material/dialog";
+import { AbsencesCreateComponent } from "../absences-create/absences-create.component";
+import { ActivatedRoute } from "@angular/router";
+import { takeUntil } from "rxjs/operators";
+import { Subject } from "rxjs";
+import { Absence } from "src/app/models/Absence.model";
+import { ApiCommunicationService } from "src/app/services/ApiCommunication.service";
+
+@Component({
+  selector: "app-emp-absences-index",
+  templateUrl: "./emp-absences-index.component.html",
+  styleUrls: ["./emp-absences-index.component.css"]
+})
+export class EmpAbsencesIndexComponent implements OnInit {
+  displayedColumns: string[] = [
+    "id",
+    "created_at",
+    "type",
+    "summary",
+    "begin",
+    "end",
+    "days",
+    "status"
+  ];
+  private _unsubscribe$: Subject<boolean> = new Subject<boolean>();
+  private _unsubscribe2$: Subject<boolean> = new Subject<boolean>();
```

Comment:
every takeUntil can use the same observable for unsubscription, please also implement OnDestroy in order to properly initiate unsubscribe logic